docs/advanced/naming.md

```diff
@@ -1,190 +1,297 @@
 
-Bad, do not use:
+![Releases using good naming schemes](/static/advanced/naming/releases-good-naming.png)
 
-```
-[Group] Anime Name - S02OVA02.mkv
-[Group] Anime Name - 18.5 [BD 1080p FLAC] [CRC].mkv
-[Group] Anime Name - 06.5 (SP) [BD 1080p FLAC].mkv
-[Group] Anime Name - Title of the Episode [CRC].mkv
-[Group] Anime Name - 17.5 (OVA) [CRC].mkv
-```
+===
 
-Good names:
+## Workflow
 
-```
-Anime Name - 06.5 - S00E04 - Title of the Episode (BD 1080p HEVC FLAC) [Dual Audio] [CRC32] [Group].mkv
-Anime Name - S00E01 - Title of the Episode (AMZN WEB-DL 1080p H.264 EAC3) [Dual Audio] [Group].mkv
-Anime Name - 18.5 - S00E05 - Title of the Episode (BD 1080p HEVC FLAC) [Dual Audio]-Group.mkv
-Anime Name (2022) - S00E01 - Title of the Episode (BD Remux 1080p HEVC FLAC) [Dual Audio]-Group.mkv
-Anime.Name.S00E09.Title.of.the.Episode.1080p.BluRay.Opus2.0.x264-Hi10P-Group.mkv
-```
+In general, a good naming scheme should:
+
+- Include the full **anime show/series** or **movie title**
+  - If there are multiple versions of the series, you must include the year within the title (e.g. [Hunter x Hunter (1999)](https://thetvdb.com/series/hunter-x-hunter) vs. [Hunter x Hunter (2011)](https://thetvdb.com/series/hunter-x-hunter-2011)). *Note that you must include the year for all movies*
+  - We recommend using names found in databases such as [TheTVDB](https://www.thetvdb.com) (for TV series) and [The Movie Database (TMDB)](https://www.themoviedb.org/?language=en-US) (for movies)
+- Include the **source type** (i.e. `BD`, `DVD`, `WEB`)
+  - Append `Remux` if your file uses a Blu-ray/DVD remux (i.e. `BD Remux` or `DVD Remux`)
+  - For `WEB` releases, you may also include the source tag (e.g. `AMZN` for Amazon, `CR` for Crunchyroll, `DSNP` for Disney+, `NF` for Netflix)
+- Include the **video resolution** and **scan type**
+  - The video resolution is based on the number of vertical pixels according to the [16:9 aspect ratio](https://wikipedia.org/wiki/16:9_aspect_ratio#Common_resolutions) (i.e. `1920x1080` = `1080`). *Note that different aspect ratios do not affect the resolution (e.g. `1440x1080` = `1080`, `1920x800` = `1080`)*
+  - Use `p` for progressive video (e.g. `1080p`, `720p`) and `i` for interlaced video (e.g. `1080i`, `720i`)
+- Include the **video and audio codec**
+  - If your file includes multiple audio tracks in different codecs, only list the default track's codec
+- Include a **releaser/group tag**
+- Limit the use of special characters or characters that need to be escaped
+  - We recommend sticking to alphanumeric characters (`A-Z`, `a-z`, `0-9`)
+  - Use `.` or `-` for delimiters
+
+### File Names
+
+In general, a good file name should:
+
+- Follow the same general guidelines for naming
+- Include the **season/episode number** (for TV series)
+  - Use `SXXEXX` format (e.g. Season 2 Episode 10 = `S02E10`)
+  - Absolute episode numbers may be used together (e.g. `S02E10 - E22`), *but never by themselves*
```

Comment:
May be worth updating this in the future to clarify on other common episode numbering.

For example, most database sites list Cardcaptor Sakura as containing 70 episodes. For ease of use, the releaser may want to use the absolute number first, which should still be fine provided they include he SxxEyy key _somewhere_ in the file name. 

Of course, tools like sonarr should actually [fix their parsing](https://github.com/Sonarr/Sonarr/issues/457#issuecomment-2130338987) so SxxEyy actually takes priority first... But still